packages/cli-plugin-metro/src/tools/loadMetroConfig.ts

```diff
@@ -97,26 +99,22 @@ export default async function loadMetroConfig(
 
   logger.debug(`Reading Metro config from ${projectConfig.filepath}`);
 
-  if (
-    !/['"']@react-native\/metro-config['"']/.test(
-      fs.readFileSync(projectConfig.filepath, 'utf8'),
-    )
-  ) {
-    logger.warn(
-      'From React Native 0.72, your metro.config.js file should extend' +
-        "'@react-native/metro-config'. Please see the React Native 0.72 " +
-        'changelog, or copy the template at:\n' +
-        'https://github.com/facebook/react-native/blob/main/packages/react-native/template/metro.config.js',
-    );
-    logger.warn('Falling back to internal defaults.');
+  if (!global.__REACT_NATIVE_METRO_CONFIG_LOADED) {
+    const warning = `
+=================================================================================================
 
-    const loadedConfig = await loadConfig(
-      {cwd: ctx.root, ...options},
-      // Provide React Native defaults on top of Metro defaults
-      getDefaultMetroConfig(ctx),
-    );
+From React Native 0.73, your project's Metro config should extend '@react-native/metro-config'
+or it will fail to build. Please copy the template at:
+https://github.com/facebook/react-native/blob/main/packages/react-native/template/metro.config.js
```

Comment:
Wonder if this should be pinned (at least to a branch) so it remains accurate in the future.

Reply:
Yeah fair. We don't have `0.73-stable` yet — and the counter-risk is pointing to a stale copy. Will keep an eye on it.